simpl/server.py

```diff
@@ -14,13 +14,111 @@
 
 """Bottle Server Module."""
 
+import copy
 import logging
 import os
+import sys
+import textwrap
 
 import bottle
 
+from simpl import config
+from simpl.utils import cli as cli_utils
+
 LOG = logging.getLogger(__name__)
 
+_fill = lambda text: textwrap.fill(text, 50)
+
+OPTIONS = [
+    config.Option(
+        '--app', '-a',
+        help=("WSGI application to load by name.\n"
+              "Ex: package.module gets the module\n"
+              "    package.module:name gets the variable 'name'\n"
+              "    package.module.func() calls func() and gets the result"),
+        group='Server Options',
+    ),
+    config.Option(
+        '--host',
+        help='Server address to bind to.',
+        default='127.0.0.1',
+        group='Server Options',
+    ),
+    config.Option(
+        '--port', '-p',
+        help='Server port to bind to.',
+        type=int,
+        default=8080,
+        group='Server Options',
+    ),
+    config.Option(
+        '--server', '-s',
+        help=('Server adapter to use. To see more, run:\n'
+              '`python -c "import bottle;print('
+              'bottle.server_names.keys())"`\n'),
+        default='xtornado',
+        group='Server Options',
+    ),
+    config.Option(
+        '--debug-server',
```

Comment:
I noticed that you changed this from `--debug\-d` to `--debug-server`. Why not just have it enabled when `--debug\-d` is supplied? I'm not sure I see the need/benefit for `--debug\-d` and `--debug-server`  being separate.

`--debug\-d` already generates gobs of data (as might be expected from a debug mode), so having it output traces in bottle also seems acceptable. _Note: for the record, --verbose is how to log heavily but not dump gobs of data out._

I do recognize the challenge of addressing the duplicate `--debug\-d` option in server.OPTIONS and log.OPTIONS if we did that. IMO, we should try to solve that and keep only one `-d` flag.